logparser/logs/micro/micro.py

```diff
@@ -0,0 +1,39 @@
+# Log Parser for RTI Connext.
+#
+#   Copyright 2016 Real-Time Innovations, Inc.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+"""Log parsing functions for Micro."""
+from __future__ import absolute_import
+import os
+import json
+
+
+def init(state):
+    """Init Micro logs."""
+    filename = "/logparser/logs/micro/error_logs.json"
+
+    with open(os.getcwd() + filename) as json_errors:
```

Comment:
Would be possible to run logparser from a different directory so the cwd is not the same as the project top level?